### ISSUE TITLE
What is the mean of "x" in loss function L_rec?

### ISSUE THREAD

**Issue by lkp520** [GuoShi28/CBDNet#43]
Hi Guo Shi
In your paper, I read
"For a batch of real images, due to the unavailability of ground-truth noise level map, only L_rec and L_TV are considered in training."
For real image, we have not clean image, so what is the mean of "x" in loss function L_rec?
Thank you for answer!
